elasticity_3d/linear_elasticity/time_dependent_examples/lame_under_internal_pressure_sin_load.py

```diff
@@ -0,0 +1,360 @@
+# import helper libraries
+import numpy as np
+import matplotlib.pyplot as plt
+import matplotlib as mpl
+#import time
+from pathlib import Path
+from matplotlib import ticker
+import matplotlib.lines as mlines
+
+# set plot options
+mpl.rcParams['mathtext.fontset'] = 'stix'
+formatter = ticker.ScalarFormatter(useMathText=True)
+formatter.set_scientific(True)
+formatter.set_powerlimits((-1,1))
+
+# import deepxde libraries
+from deepxde import backend as bkd
+import deepxde as dde
+from deepxde import config
+
+
+# import computational mechanics libraries
+from utils.geometry.custom_geometry import GmshGeometry3D
+from utils.geometry.gmsh_models import Geom_step_to_gmsh
+from utils.elasticity import elasticity_utils
+from utils.elasticity.elasticity_utils import pde_mixed_3d_time, get_tractions_mixed_3d_spacetime, problem_parameters
+from utils.postprocess.elasticity_postprocessing import solutionFieldOnMeshToVtkSpaceTime
+from utils.postprocess.save_normals_tangentials_to_vtk import export_normals_tangentials_to_vtk
+from utils.geometry.geometry_utils import polar_transformation_3d_spherical, calculate_boundary_normals_3D
+from utils.geometry.geometry_time import ModifiedGeometryXTime
+
+'''
+@author: tsahin
+
+3D lame problem with time.  
+
+D. Schillinger, J. A. Evans, F. Frischmann, R. R. Hiemstra, M.-C. Hsu, T. J. Hughes, A
+collocated c0 finite element method: Reduced quadrature perspective, cost comparison
+with standard finite elements, and explicit structural dynamics, International Journal
+for Numerical Methods in Engineering 102 (3-4) (2015) 576–631.
+'''
+
+path_to_step_file = str(Path(__file__).parent.parent.parent)+f"/step_files/lame_3d.stp"
```

Comment:
When I tried to run this example on my local machine, I ran into some issues, when executing this script from the folder it's contained in (because then `__file__` evaluates to `"./lame_under_internal_pressure_sin_load.py"`, so repeatedly accessing `.parent` results just in `"./"`. Therefore I propose the following small addition:
```suggestion
path_to_step_file = str(Path(__file__).resolve().absolute().parent.parent.parent)+f"/step_files/lame_3d.stp"
```
I guess your version works fine, as long as this script is invoked from a level higher than where the step files are being kept, i.e., the repository root directory.